.github/workflows/git-secrets.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   # This workflow contains a single job called "main"
   git-secrets:
     # The type of runner that the job will run on
-    runs-on: ubuntu-18.04
+    runs-on: macos-latest
```

Comment:
can you clarify why you changed it? The OS can impact the way we sigh the code by codesigning command during new version release, it is sensitive